docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+         max_slots: 1
+
+If a user tries to set a notebook priority to 10 when the limit is 15, the request will fail, and
+the system will display an error message.
+
+Limit Maximum GPU Usage per Experiment
+======================================
+
+The following configuration policy example limits GPU usage per experiment. This is configured in
+the Config Policies > Experiments tab.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       max_slots: 4
+
+Priority Limits
+===============
+
+You can set a different priority limit for experiments and tasks.
+
+The following policy is configured in the Config Policies > Experiments tab.
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       priority_limit: <experiment limit>
+
+The following policy is configured in the Config Policies > Tasks tab.
+
+-  Tasks
+
+.. code:: yaml
+
+   constraints:
+     resources:
+       priority_limit: <task limit>
+
+Invariant Configs
+=================
+
+Invariant configs are applied to all workloads in a given scope (global or workspace) and merged
+with user-submitted configurations.
+
+**Set Default Priority for All Experiments**
+
+The following configuration policy example sets a default priority for all experiments:
+
+.. code:: yaml
+
+   invariant_config:
+     resources:
+       priority: 5
+
+**Example Invariant Config for Bind Mounts and Environment Variables**
+
+This example shows how to apply default configurations across all workloads using bind mounts and
+environment variables and is set in the Experiments tab:
+
+.. code:: yaml
+
+   invariant_config:
+     bind_mounts:
+       - host_path: "/etc/bindmounts"
+         container_path: "/etc/bindmounts"
+         read_only: true
+     data:
+       datapoint1: value1
+     debug: true
+     environment:
+       environment_variables:
+         cpu:
+           - cpuval=originalcpu
+
+When a user submits a workload, these settings will be applied in addition to (or overriding) the
+user's settings.
+
+.. warning::
+
+   Do not set both constraints and configs for the same field within the same workload type and
+   scope. It may lead to unpredictable behavior.
+
+.. important::
+
+   -  Existing workloads are not impacted by new or updated policies.
+   -  Updating config policies replaces the entire set of invariant configs and constraints.
+
+*******************
+ Priority Override
+*******************
+
+If a global or workspace-level policy sets a default priority, it can be overridden using the
+`SetJobPriority` API endpoint. However, the new priority must respect the constraints defined in the
+applicable policies.
+
+.. note::
+
+   Higher values indicate higher priority in Kubernetes resource managers, while lower values
+   indicate higher priority in Agent resource managers.
+
+****************
+ Best Practices
+****************
+
+-  Start with cluster-wide policies for broad governance.
+-  Use workspace and project-level policies for more granular control.
```

Comment:
We do not have project-level policies, just workspaces. If I recall correctly, a workspace can have multiple projects right?

Reply:
This is correct, workspaces can have multiple projects (in which you can launch experiments), but config policies are only enforced at the workspace level!

Reply:
true, good catch!